scripts/restore.sh

```diff
@@ -194,6 +194,7 @@ restore_pane() {
 		else
 			new_session "$session_name" "$window_number" "$window_name" "$dir" "$pane_index"
 		fi
+    tmux select-pane -t "$session_name":"$window_number"."$pane_index" -T "${pane_title}"
```

Comment:
Instead of naming the pane with a standalone command, can we somehow set the correct pane_title when the pane is created?